sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/amd64_defconfig-6.6

```diff
@@ -11,6 +11,8 @@ CONFIG_ACPI_HOTPLUG_MEMORY=y
 CONFIG_ACPI_IPMI=m
 CONFIG_ACPI_PCI_SLOT=y
 CONFIG_ACPI_PROCESSOR_AGGREGATOR=y
+CONFIG_AGP=y
+CONFIG_AGP_INTEL=y
```

Comment:
AGP is a bus that predates PCI express, and hasn't been in use for 20 years. I would not enable it in our config.

Reply:
I did enable it 'just in case',  because I saw this documentation here: https://github.com/torvalds/linux/blob/master/drivers/char/agp/Kconfig#L74

Sure, will disable the config if the module is not used or useful. Also saw that Ubuntu and other distros have the same configs enabled, but probably for legacy reasons.